lib/rpmrc.c

```diff
@@ -736,6 +736,16 @@ static rpmRC rpmPlatform(rpmrcCtx ctx, const char * platform)
 }
 
 
+#	if defined(__linux__) && defined(__x86_64__)
```

Comment:
Hmm, shouldn't the #if match up with the one around is_ryzen or be more general?

Reply:
You're right, this was accidentally broken when I rebased it again...

Reply:
Wait nope, there's a `cpuid()` implementation for i386 right below...

Reply:
OK , there's a #	if defined(__linux__) && defined(__i386__) branch just below with its own implementation of cpuid.